iam/x509-identity-mgmt/js/config/default.conf

```diff
@@ -325,7 +325,7 @@ kafka.topic.acks:integer=-1
 # ----------------------------------------
 # X509IDMGMT_KAFKA_TOPIC_AUTO_OFFSET_RESET
 # ----------------------------------------
-kafka.topic.auto.offset.reset:string=latest
+kafka.topic.auto.offset.reset:string=earliest
```

Comment:
I don't see a problem of using 'latest' once you are implementing a cache and go to the device manager when a miss happens, but there is no problem to use 'earliest' too. This property only impacts the first time you connect with Kafka.